OakCityV2/Job.m

```diff
@@ -0,0 +1,35 @@
+//
+//  DataObject.m
+//  OakCityV2
+//
+//  Created by Josh Green on 6/29/15.
+//  Copyright (c) 2015 Josh Green. All rights reserved.
+//
+
+#import "Job.h"
+
+@interface Job ()
+/*
+@property (nonatomic, readwrite) NSString *title;
+@property (nonatomic, readwrite) NSString *company;
+@property (nonatomic, readwrite) NSString *desc;
+@property (nonatomic, readwrite) NSString *url;
+@property (nonatomic, readwrite) NSString *logo;
+*/
+@end
```

Comment:
Yeah, this is what you want.  You want to redeclare these properties as readwrite in the private class extension.  Because it's in the .m file, it only applies within this file.  Other objects in the system only see the declaration in the .h, so it'll be readonly externally.